groovy/postInit/chemistry/inorganic_chemistry/elements/p_block/group18/AirDistillation.groovy

```diff
@@ -14,411 +14,571 @@ FBR = recipemap('fixed_bed_reactor')
+    .fluidOutputs(fluid('compressed_helium_neon_concentrate') * 250)
+    .duration(500)
     .EUt(Globals.voltAmps[3])
     .buildAndRegister()
 
-//REFLUXED DISTILLATION
 BATH_CONDENSER.recipeBuilder()
-    .fluidInputs(fluid('cold_waste_gaseous_nitrogen') * 8)
-    .fluidOutputs(fluid('liquid_waste_nitrogen') * 1)
-    .duration(1)
+    .fluidInputs(fluid('liquid_nitrogen') * 125)
+    .fluidOutputs(fluid('nitrogen') * 8000)
+    .fluidInputs(fluid('compressed_helium_neon_concentrate') * 250)
+    .fluidOutputs(fluid('treated_liquid_nitrogen') * 125)
+    .fluidOutputs(fluid('helium_neon_mixture') * 69)
+    .duration(100)
+    .EUt(Globals.voltAmps[1])
     .buildAndRegister()
 
-SINGLE_COLUMN_CRYOGENIC_DISTILLATION_PLANT.recipeBuilder()
-    .circuitMeta(2)
-    .fluidInputs(fluid('liquid_deoxygenated_argon') * 200)
-    .fluidInputs(fluid('argon') * 400)
-    .fluidInputs(fluid('liquid_waste_nitrogen'))
-    .fluidOutputs(fluid('liquid_argon_product') * 250)
-    .fluidOutputs(fluid('cold_waste_gaseous_nitrogen') * 24)
-    .duration(83)
+PSA.recipeBuilder()
+    .notConsumable(metaitem('zeolite_membrane'))
+    .fluidInputs(fluid('helium_neon_mixture') * 69)
+    .fluidOutputs(fluid('helium') * 54)
+    .fluidOutputs(fluid('neon') * 15)
+    .duration(500)
     .EUt(Globals.voltAmps[3])
     .buildAndRegister()
-
-//STARTUP SUBCOOLING
-HEAT_EXCHANGER.recipeBuilder()
-    .fluidInputs(fluid('oxygen_rich_liquid') * 75)
-    .fluidInputs(fluid('liquid_nitrogen') * 150)
-    .fluidOutputs(fluid('subcooled_oxygen_rich_liquid') * 75)
-    .fluidOutputs(fluid('nitrogen') * 9600)
+    
+//KRYPTON & XENON
+SINGLE_COLUMN_CRYOGENIC_DISTILLATION_PLANT.recipeBuilder()
+    .fluidInputs(fluid('untreated_liquid_oxygen') * 105)
+    .fluidOutputs(fluid('krypton_xenon_rich_liquid')) // Contains 0.0264 Kr and 0.00216 Xe gas
+    .fluidOutputs(fluid('treated_liquid_oxygen') * 105)
     .duration(5)
+    .EUt(Globals.voltAmps[1])
     .buildAndRegister()
 
-//NORMAL SUBCOOLING
-HEAT_EXCHANGER.recipeBuilder()
-    .fluidInputs(fluid('oxygen_rich_liquid') * 75)
-    .fluidInputs(fluid('liquid_argon_product') * 5)
-    .fluidOutputs(fluid('subcooled_oxygen_rich_liquid') * 75)
-    .fluidOutputs(fluid('partially_liquefied_argon') * 18)
+BR.recipeBuilder()
+    .fluidInputs(fluid('krypton_xenon_rich_liquid'))
+    .fluidInputs(fluid('oxygen') * 10)
+    .fluidOutputs(fluid('combusted_krypton_xenon_concentrate') * 48)
     .duration(5)
+    .EUt(Globals.voltAmps[1])
     .buildAndRegister()
 
-PHASE_SEPARATOR.recipeBuilder()
-    .fluidInputs(fluid('partially_liquefied_argon') * 450)
-    .fluidOutputs(fluid('argon') * 400)
-    .fluidOutputs(fluid('liquid_argon_product') * 50)
-    .duration(29)
+PSA.recipeBuilder()
+    .notConsumable(metaitem('zeolite_membrane'))
+    .fluidInputs(fluid('combusted_krypton_xenon_concentrate') * 48)
+    .fluidOutputs(fluid('purified_krypton_xenon_concentrate') * 48)
+    .duration(100)
```

Comment:
The duration on this recipe is 20 times longer than the previous one, even though it's one-to-one.